ui/widgets/nuxeo-directory-suggestion.js

```diff
@@ -255,7 +261,7 @@ import { escapeHTML } from './nuxeo-selectivity.js';
           directoryName: this.directoryName,
           dbl10n: this.dbl10n,
           canSelectParent: this.canSelectParent,
-          localize: true,
+          localize: this.localize,
```

Comment:
This property is not defined (see Sam's comment above), which means we are changing the default behavior.

Reply:
property is now defined, I've fixed the comment